packages/api/src/controllers/deliberations/v2/mod.rs

```diff
@@ -48,17 +43,54 @@ impl DeliberationController {
             roles,
         }: DeliberationCreateRequest,
     ) -> Result<Deliberation> {
-        // TODO: implement temporary and create
-        todo!()
+        if started_at >= ended_at {
+            return Err(ApiError::ValidationError(
+                "started_at should be less than ended_at".to_string(),
+            )
+            .into());
+        }
+
+        let deliberation = self
+            .repo
+            .insert(
+                org_id,
+                started_at,
+                ended_at,
+                project_area,
+                title,
+                description,
+            )
+            .await?;
+
+        // TODO: Add steps
+
+        Ok(deliberation)
     }
 
     pub async fn query(
         &self,
         org_id: i64,
         DeliberationQuery { size, bookmark }: DeliberationQuery,
-    ) -> Result<QueryResponse<Deliberation>> {
-        // TODO: impelement query
-        todo!()
+    ) -> Result<QueryResponse<DeliberationSummary>> {
+        let mut total_count: i64 = 0;
+        let query: Vec<DeliberationSummary> = Deliberation::query_builder()
```

Comment:
```suggestion
        let items: Vec<DeliberationSummary> = Deliberation::query_builder()
```
it will be simply populated in line 92.